streaming/streaming.go

```diff
@@ -446,20 +479,62 @@ func getStreamingConfig(sourceProfile profiles.SourceProfile, targetProfile prof
 	}
 }
 
+func CreateStreamingConfig(pl profiles.DataShard) StreamingCfg {
+	//create dataflowcfg from pl receiver object
+	dataflowCfg := DataflowCfg{}
```

Comment:
Do any inputs need to be validated here ?

Reply:
Inputs are validated in the `VerifyAndUpdateCfg` method which is called after this one.
Code snippet - 

```
streamingCfg := streaming.CreateStreamingConfig(*p)
//verify the CFG and update it with HB defaults
err := streaming.VerifyAndUpdateCfg(&streamingCfg, targetProfile.Conn.Sp.Dbname)
```